src/components/input/index.tsx

```diff
@@ -124,7 +129,15 @@ const Input = forwardRef<HTMLInputElement, Props>(
         hasError={hasError}
         disabled={disabled}
         required={required}
-        onChange={onChange}
+        onChange={(e: ChangeEvent<HTMLInputElement>) => {
+          e.persist()
+          setInputEvent({
+            ...e,
+            target: {
+              ...e.target,
+            },
+          })
```

Comment:
This may look weird but it was the only workaround to make the unit test pass. For some reason, if I pass the `e` directly, the `e.target.value` was passed as empty when in reality it contains value. This caused the test to fail.
This only happend in testing environment.

Reply:
this sounds _very_ odd. I'd prefer to understand what is going on exactly before merging with this workaround